Podfile

```diff
@@ -0,0 +1,14 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '9.0'
```

Comment:
No iOS o melhor geralmente é evitar usar bibliotecas quando necessario ou se for pequena puxar o codigo pra dentro do app. Se o dono da biblioteca nao atualizar a versao do swift (o que acontece as vezes) e o projeto ficar muito acoplado com a biblioteca ele pode comecar a nao compilar e vc ser forcado a atualizar a lib manualmente.